go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/aws/aws-sdk-go v1.15.39
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/cenkalti/backoff v2.0.0+incompatible
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
is this library well-known in golang?
I'm always skeptic for such library after reading this story https://hackernoon.com/im-harvesting-credit-card-numbers-and-passwords-from-your-site-here-s-how-9a8cb347c5b5